artifacts/reviews/archscope-evidence-p0-manual-review.md

```
@@ -0,0 +1,41 @@
+# Archscope Evidence P0 Manual Review
+
+## Scope
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)